packages/preprocessors/compiler.js

```diff
@@ -182,18 +182,23 @@ function replaceSlashes(str) {
 	return str.replace(/\\+/g, '/').replace(/\/{2,}/g, '/');
 }
 
-function buildJsio(opts, callback) {
-	function getJsioSrc() {
-		var src = jsio.__jsio.__init__.toString(-1);
+exports.getJsioSrc = function(includeJsio) {
```

Comment:
it's a bit odd to have a function `getJsioSrc` that has a parameter for not including the jsio src, both `getJsioSrc` and `getPathJS` are exported, can we just remove the parameter and have callers use `getPathJS` if that's all they want?